pkgs/top-level/aliases.nix

```diff
@@ -215,11 +215,8 @@ mapAliases ({
   elixir_ls = elixir-ls; # Added 2023-03-20
 
   # Emacs
-  emacs28NativeComp = emacs28; # Added 2022-06-08
-  emacs28Packages = emacs28.pkgs; # Added 2021-10-04
-  emacs28WithPackages = emacs28.pkgs.withPackages; # Added 2021-10-04
```

Comment:
What should we do with these versioned aliases?

I dislike the idea that bumping these versioned every time the default Emacs version is bumped because this will keep these aliases forever.

Reply:
They are here just because we were maintaining two versions at the same time.

There is no need to bump them.